modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogColumnCollation.java

```diff
@@ -70,4 +70,17 @@ public boolean asc() {
     public boolean nullsFirst() {
         return nullsFirst;
     }
+
+    /** Writes collation to byte. */
+    static byte pack(CatalogColumnCollation collation) {
+        return  (byte) ((collation.asc() ? 1 : 0) | (collation.nullsFirst() ? 2 : 0));
+    }
+
+    /** Reads collation from byte. */
+    static CatalogColumnCollation unpack(byte collationBits) {
+        boolean asc = (collationBits & 1) == 1;
+        boolean nullsFirst = (collationBits & 2) == 2;
```

Comment:
Let's convert magic numbers to constants.

Reply:
Added constants for bit positions.